functions/calculate-score/v1.js

```diff
@@ -0,0 +1,40 @@
+'use strict'
+
+const moment = require('moment')
+const scoreCalculationDataService = require('./score-calculation.data.service')
+const checkWindowDataService = require('./check-window.data.service')
+
+const v1 = {
+  process: async function (context) {
+    await handleCalculateScore(context)
+  }
+}
+
+async function handleCalculateScore (context) {
+  const liveCheckWindow = await checkWindowDataService.sqlFindCalculationPeriodCheckWindow()
+
+  // Terminate execution if a check window is not within the calculation period
+  if (!liveCheckWindow || !liveCheckWindow.id) {
+    context.log(`calculate-score: Live check window not found`)
+    return
+  }
+
+  // Terminate execution if relevant check window is finalised
+  if (liveCheckWindow.complete) {
+    context.log(`calculate-score: National score has been calculated for check window id ${liveCheckWindow.id}`)
```

Comment:
could you change the wording slightly to 'Check window calculation already complete.  Bypassing calculation run'

Reply:
@GuyHarwood done